CONTRIBUTING.rst

```diff
@@ -438,6 +488,8 @@ above?
 All the above are not rules, but merely some guidelines/suggestions on what we should expect
 about backports.
 
+Backports should be **squashed** (rather than **merged**), as doing so preserves the original PR author correctly.
```

Comment:
Did you mean “backport PRs”? Not just commits that exist in them?

Reply:
Yes that is exactly what I mean, updated.

Reply:
I appeared to me that this could be enforced for stable branches on the ruleset level.